app/presenters/sufia/select_type_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module Sufia
+  class SelectTypePresenter
+    def initialize(concern)
+      @concern = concern
+    end
+
+    attr_reader :concern
+
+    def icon_class
+      translate('icon_class')
+    end
+
+    def description
+      translate('description')
+    end
+
+    def name
+      translate('name')
+    end
+
+    private
+
+      def object_name
+        @object_name ||= concern.model_name.i18n_key
+      end
+
+      def translate(key)
+        defaults = []
+        defaults << :"sufia.select_type.#{object_name}.#{key}"
+        defaults << :"sufia.select_type.#{key}"
+        defaults << ''
+        I18n.t(defaults.shift, default: defaults)
```

Comment:
I don't think we should test the behavior of another gem.  The line is covered and we are testing that all the public methods work correctly